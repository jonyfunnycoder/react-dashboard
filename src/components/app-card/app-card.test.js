import React from 'react';
import renderer from "react-test-renderer";
import AppCard from "./app-card";

test('renders correctly ', () => {
    const tree = renderer
        .create(
            <AppCard>
                <h1>Just a card</h1>
            </AppCard>
        )
        .toJSON();

    expect(tree).toMatchSnapshot();
});
