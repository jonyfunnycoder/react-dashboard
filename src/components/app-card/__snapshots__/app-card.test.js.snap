// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly  1`] = `
<div
  className="app-card card"
>
  <div
    className="card-body"
  >
    <h1>
      Just a card
    </h1>
  </div>
</div>
`;
